### PR TITLE
`moment-locales-webpack-plugin`: upgrade webpack types to v5

### PR DESCRIPTION
**Pull request by xarety**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/moment-locales-webpack-plugin/tsconfig.json [hunk 1]
```diff
@@ -1,24 +1,28 @@
...
+        ],
         "paths": {
             "webpack": [
-                "webpack/v4"
+                "./node_modules/webpack"
             ],
             "tapable": [
-                "tapable/v1"
+                "./node_modules/tapable"
             ]
```
Comment (rbuckton) [inline types/moment-locales-webpack-plugin/tsconfig.json:21]: Do you still need these entries if you're leveraging `package.json` dependencies for these imports?

Comment (xarety) [reply types/moment-locales-webpack-plugin/tsconfig.json:21]: I think so; I did the same as other packages did. e.g. https://github.com/DefinitelyTyped/DefinitelyTyped/pull/56391

Comment (andrewbranch) [reply types/moment-locales-webpack-plugin/tsconfig.json:21]: Can we try without and see if CI passes? I just looked through a bunch of other packages with dependencies and I’m pretty sure this shouldn’t be needed.

Comment (andrewbranch) [reply types/moment-locales-webpack-plugin/tsconfig.json:21]: Ah wait, the issue is that there’s still a `types/webpack` and `types/tapable` folder, so this _is_ needed in order not to use those.

**Review by andrewbranch — APPROVED** [review]

**Comment by xarety** [comment]
Ready to merge

**Pull request merged**